src/sls_core.js

```diff
@@ -247,12 +247,14 @@ function createSingleSlide(deck, insightDeck, slideLayout) {
     slideBody.setText(body);
   }
 
-  const imageShapesArray = documentProperties.getProperty('IMAGE_SHAPES')
-      .split(',')
-      .map((item) => item.trim());
-  const imageRangesArray = documentProperties.getProperty('IMAGE_RANGES')
-      .split(',')
-      .map((item) => item.trim());
+  const imageShapesArray =
+      (documentProperties.getProperty('IMAGE_SHAPES') || '')
+          .split(',')
+          .map((item) => item.trim());
+  const imageRangesArray =
+      (documentProperties.getProperty('IMAGE_RANGES') || '')
+          .split(',')
+          .map((item) => item.trim());
 
   if (imageShapesArray && imageShapesArray.length > 0) {
```

Comment:
This is always true, since `imageShapesArray` is always at least an array with one empty string. 

If you skipped the `||` and used optional chaining (`?.`) in the assignment, this would end up undefined if the property doesn't exist, you'd skip the loop and be able to remove the if on line 264.

I noticed this pattern further down too.